services/simcore/docker-compose.yml

```diff
@@ -532,6 +532,31 @@ services:
           memory: 128M
           cpus: '0.1'
 
+  efs-guardian:
+    hostname: "{{.Service.Name}}"
+    deploy:
+      replicas: ${SIMCORE_EFS_GUARDIAN_REPLICAS}
```

Comment:
@mrnicegyu11 I would not use ENV for replicas but just write then in `docker-compose.<deployment>.yaml`. Do you have an opinion here?

Reply:
@YuryHrytsuk The issue here is that I cannot differentiate between AWS deployments since you have only one file for that. Therefore I need to differentiate via ENV vars.